KiBOM_CLI.py

```diff
@@ -47,7 +61,10 @@ def say(*arg):
 
 def isExtensionSupported(filename):
     result = False
-    extensions = [".xml",".csv",".txt",".tsv",".html"]
+    if xlsxwriter_available:
```

Comment:
I would have a default extensions array, and then optionally append '.xlsx' if xlsxwriter_available is True